aksel.nav.no/website/sanity/schema/documents/grunnleggende/endringsloggartikkel.tsx

```diff
@@ -0,0 +1,91 @@
+import { defineField, defineType } from "sanity";
+import { FileCodeIcon, FileImageIcon, FileTextIcon } from "@navikt/aksel-icons";
+import { capitalize } from "@/utils";
+import SanityTabGroups from "../presets/groups";
+import { hiddenFields } from "../presets/hidden-fields";
+import { sanitySlug } from "../presets/slug";
+import { titleField } from "../presets/title-field";
+
+export const EndringsloggArtikkel = defineType({
+  title: "Endringsloggartikkel",
+  name: "ds_endringsloggartikkel",
```

Comment:
`ds_endringsloggartikkel` needs to be added to the `filtered` list in `structure/index.tsx`-file to avoid this 
<img width="329" alt="Screenshot 2025-05-16 at 11 15 41" src="https://github.com/user-attachments/assets/c85b5437-b290-47c5-a124-174acdbe9aa4" />